test/framework/alertmanager.go

```diff
@@ -217,10 +217,7 @@ func amImage(version string) string {
 func (f *Framework) WaitForAlertmanagerInitialized(ns, name string, amountPeers int, forceEnableClusterMode bool) error {
 	var pollError error
 	err := wait.Poll(time.Second, time.Minute*5, func() (bool, error) {
-		amStatus, err := f.GetAlertmanagerStatus(ns, name)
-		if err != nil {
-			return false, err
-		}
+		amStatus := f.GetAlertmanagerStatus(ns, name)
```

Comment:
As written in the issue, we should return false to retry later and the returned error needs to be saved in case the wait loop fails eventually. 

```suggestion
		amStatus, err := f.GetAlertmanagerStatus(ns, name)
		if err != nil {
			pollError := fmt.Errorf("failed to query Alertmanager: %s", err)
			return false, nil
		}
```